src/styles/blocks/nav-menu.scss

```
@@ -0,0 +1,49 @@
+@use "../utils/extends";
+@use "../utils/variables" as *;
+
+.nav__list {
+  @extend %resetMarginPadding ;
+  display: flex;
+  list-style: none;
+  flex-wrap: wrap;
+  align-items: center;
```

```suggestion
.nav {
  &__list {
  @extend %resetMarginPadding ;
  display: flex;
  list-style: none;
  flex-wrap: wrap;
  align-items: center;
```
Use nesting correctly